source/installing-docker.rst

```diff
@@ -16,9 +18,47 @@ To create the container, start it, and add the default user:
     $ docker create --name pufferpanel -p 8080:8080 -p 5657:5657 -v pufferpanel-config:/etc/pufferpanel -v /var/lib/pufferpanel:/var/lib/pufferpanel -v /var/run/docker.sock:/var/run/docker.sock --restart=on-failure pufferpanel/pufferpanel:latest
     $ docker start pufferpanel
     $ docker exec -it pufferpanel /pufferpanel/pufferpanel user add
-    
+
 And you're done. Your panel is now accessible at http://localhost:8080
 
+----
+
+**Via Docker Compose**
+
+You will need the Docker Compose plugin installed on your system.
+
+Create the Pufferpanel directory: 
+
+.. code-block:: bash
+
+    $ mkdir -p /var/lib/pufferpanel
+
+Create the following ``docker-compose.yml`` file which defines the Pufferpanel service, including its ports and volume mappings. This is equivalent to the command line setup above. 
+
+.. code-block:: yml
+
+    version: '3.2'
+    services:
+      pufferpanel:
+        image: pufferpanel/pufferpanel:latest # See "Tags" for other images
+        restart: on-failure
+        ports:
+          - 8080:8080 # XXXX:8080, where port XXXX on the outside (host) will be sent to 8080 inside the container
+          - 5657:5657 # YYYY:5657
+        volumes:
+          - ./pufferpanel-config:/etc/pufferpanel
```

Comment:
You can use a docker volume here directly with compose, and thats better than another host bind.

Reply:
Configuration directory binding changed from host bind to a named docker volume created with the compose file.